de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopPresenterImpl.java

```diff
@@ -588,9 +588,9 @@ public void saveUserSettings(final UserSettings value,
                                  final boolean updateSilently) {
         final RuntimeCallbacks.SaveUserSettingsCallback callback = new RuntimeCallbacks.SaveUserSettingsCallback(value,
                                                                                userSettings,
-                                                                               announcer,
+                                                                               announcer,this,
                                                                                appearance,
-                                                                               updateSilently);
+                                                                               updateSilently,userSessionService);
```

Comment:
@sri2k1us, when making changes like this, within the scope of a PR, it is messy to make new commits for small changes like this.

You could easily merge the commits and force push back to the branch to clean up the history.